.github/workflows/setupWlsAks.yml

```diff
@@ -5,16 +5,22 @@ on:
     inputs:
       disambiguationSuffix:
         description: "Included in names to disambiguate. Get from another pipeline execution"
+        default: "wls"
+        required: true
+      deleteResource:
+        description: "True to delete all the Azure resources after the deployments. False to keep Azure resources."
+        default: "true"
         required: true
   repository_dispatch:
+  schedule:
+    - cron: '0 0 13 * *' # run the workflow at the end of 21th monthly.
 
 # All of the secrets in the env section are required. Run setup.sh to
 # populate the git repository running this workflow with the necessary secrets.
     
 env:
-  aksRepoUserName: oracle
-  aksRepoBranchName: 278a527cb7d1eb44a5a65ccc02b6837fe42967db
-  aksClusterConfigMapName: wlsd-wdt-config-map-${{ github.event.inputs.disambiguationSuffix }}
+  aksRepoUserName: ${{ secrets.AKS_REPO_USER_NAME }}
+  aksRepoBranchName: 2024-03-13-2-Q1
```

Comment:
Update to `2024-03-21-1-Q1`.

Reply:
Resolved in https://github.com/Azure-Samples/cargotracker-wls-aks/pull/7/commits/635c0079fb9d5c31d681d07e6cc147bb446393c1